scipy/spatial/src/distance_wrap.c

```diff
@@ -32,6 +32,16 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* avoid unknown pragma warning in clang which may be used by developers */
+#ifndef __clang__
```

Comment:
No not used, just googled when the optimize pragma was introduced and typed the code in this comment box.